googleapis/google-cloud-node
# migrate code from googleapis/nodejs-resource-settings

- feat: initial stub of library
- feat: generate initial samples
- feat: add initial samples and tests (#2)
- chore(deps): update dependency mocha to v9 (#4)
- chore(deps): update dependency sinon to v11 (#5)
- chore: release 1.0.0 (#1)
- chore: release 1.0.0 (#6)
- fix(deps): google-gax v2.17.1 (#8)
- chore: release 1.0.1 (#9)
- build(nodejs): correct package name for npm (#14)
- chore(deps): update dependency @types/mocha to v9 (#15)
- chore(nodejs): update client ref docs link in metadata (#18)
- fix(deps): google-gax v2.24.1 (#20)
- chore: release 1.0.2 (#21)
- feat: turns on self-signed JWT feature flag (#22)
- chore: release 1.1.0 (#23)
- fix: flag module as GA (#25)
- chore: release 1.1.1 (#26)
- chore: disable renovate dependency dashboard (#1194) (#27)
- fix(build): switch primary branch to main (#28)
- chore: release 1.1.2 (#29)
- build(node): update deps used during postprocessing (#1243) (#36)
- chore: fix the wrong post processor image (#39)
- chore(deps): update dependency @types/node to v16 (#38)
- chore(deps): update dependency sinon to v12 (#40)
- docs(samples): add example tags to generated samples (#41)
- build: add generated samples to .eslintignore (#42)
- docs(node): support "stable"/"preview" release level (#1312) (#45)
- chore: add api_shortname and library_type to repo metadata (#44)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#47)
- test(nodejs): remove 15 add 16 (#1322) (#49)
- build: update copyright year to 2022 (#51)
- chore: update v2.12.0 gapic-generator-typescript (#53)
- chore(deps): update dependency sinon to v13 (#55)
- docs(nodejs): version support policy edits (#1346) (#57)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#58)
- chore: update v2.14.2 gapic-generator-typescript (#62)
- build(node): update client library version in samples metadata (#1356) (#69)
- chore(deps): update dependency sinon to v14 (#72)
- build!: update library to use Node 12 (#74)
- chore(main): release 2.0.0 (#75)
- chore(deps): update dependency jsdoc-fresh to v2 (#77)
- chore(deps): update dependency jsdoc-region-tag to v2 (#78)
- fix(docs): describe fallback rest option (#79)
- chore(main): release 2.0.1 (#80)
- chore(deps): update dependency linkinator to v4 (#84)
- fix: better support for fallback mode (#85)
- fix: change import long to require (#86)
- fix: do not import the whole google-gax from proto JS (#1553) (#89)
- fix: allow passing gax instance to client constructor (#90)
- chore(main): release 2.0.2 (#87)
- fix: preserve default values in x-goog-request-params header (#92)
- test: use fully qualified request type name in tests (#95)
- fix(deps): use google-gax v3.5.2 (#99)
- chore(deps): update dependency @types/node to v18 (#97)
- chore(deps): update dependency jsdoc to v4 (#101)
- fix: regenerated protos JS and TS definitions (#102)
- chore(main): release 2.0.3 (#93)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
